OPCB/functions/tankspawner/spawn_tank_server.sqf

```diff
@@ -1,12 +1,11 @@
 params ["_vehicle","_isAttack"];
 _staticType = ["rhs_Metis_9k115_2_vmf","rhs_Kornet_9M133_2_vmf","RHS_Stinger_AA_pod_D","RHS_M2StaticMG_D","RHS_M2StaticMG_MiniTripod_D","RHS_TOW_TriPod_D","RHS_MK19_TriPod_D","B_Mortar_01_F","B_Static_Designator_01_F"];
-_tankpos = markerPos "tank_spawner";
+_tankpos = getMarkerPos "tank_spawner";
+
+_tank = createVehicle [_vehicle, _tankpos, [], 0 , "CAN_COLLIDE"];
+_tank setdir (markerDir "tank_spawner");
```

Comment:
hopefully we've set the marker dir properly in the editor :D

Reply:
that's a different PR :P